src/common/utils.sh

```diff
@@ -83,3 +83,32 @@ function do_curl {
     return 1
   fi
 }
+
+#######################################
+# Retries a command until it succeeds, with an increasing delay between each attempt
+# Globals:
+#   None
+# Arguments:
+#   $@: command to retry
+# Returns:
+#   exitCode: the exit code of the last attempt
+#######################################
+function retry_with_backoff {
```

Comment:
Where would this function be used?

Reply:
It would be used in the instantiate_fabric_chaincode function, where it is currently being invoked by itself:
https://github.com/IBM-Blockchain-Starter-Kit/build-lib/blob/65d428682af2b27de1cda473ea68600ac166b469/src/common/blockchain.sh#L234-L240

Reply:
There are more uses coming as well, e.g. waiting for a peer to start/stop